impl/src/main/java/jakarta/faces/component/UIColumn.java

```diff
@@ -16,6 +16,8 @@
 
 package jakarta.faces.component;
 
+import com.sun.faces.api.component.UIColumnImpl;
```

Comment:
Is this intentional or temporary?

Reply:
Definitely temporary. I'd like to have the components working first, and then in a second pass hide those *impl classes.